corehq/pillows/mappings/case_search_mapping.json

```diff
@@ -34,7 +34,8 @@
                         "exact": {
                             "index": "not_analyzed",
                             "type": "string",
-                            "null_value": ""
+                            "null_value": "",
+                            "ignore_above": 8191
```

Comment:
do you also need it on the main `value` field?

Reply:
I don't believe so. Specifically the error message referenced this field, not case properties as a whole `Document contains at least one immense term in field="case_properties.value.exact"`, which makes sense because each field is indexed separately rather than as a whole, so it's specifically this string one that is having the issue, but as with the other question above, I will make sure to confirm on staging.